src/components/sbb-autocomplete/sbb-autocomplete.stories.tsx

```diff
@@ -129,22 +129,22 @@ const defaultArgs: Args = {
   floatingLabel: false,
 };
 
-const withGroupsDefaultArgs = {
+const withGroupsDefaultArgs: Args = {
   ...defaultArgs,
 
   // Option group args
   disableGroup: false,
 };
 
-const defaultDecorator = [
+const defaultDecorator: Decorator[] = [
```

Comment:
either write it as decorators or only assign a single decorator, otherwise it's confusing to read (?)
```suggestion
const defaultDecorators: Decorator[] = [
```